### ISSUE TITLE
Data Exploration Expansion

### ISSUE THREAD

**Issue by koay9f** [ORNL-AMO/AMO-Tools-Desktop#4609]
* In data visualization tab need to have a time series line graph (histogram, scatter, time series)
* HOWEVER - Excel wipes the seconds from the time stamp when you save as csv. making anything smaller than minute data unusable.
* Would like to be able to intake excel.  that might solve the above problem.
*Also some data has date & time as two separate columns.  need to be able to handle that.

**Comment by koay9f** [comment]
So this got sneaked into the compressed-air branch I was looking at so I was able to look at it a bit.
Excel is working great, but it might be nice if we could specify which worksheet we wanted to look at.  If not really feasible, let me know.

Turns out this didn't really fix the date time thing.  It looks like Excel will at least save the seconds, but if it is not formatted with a custom date-time format, then it still gets screwy.
Is there any way we can take the date-time serial number (instead of the formatted date time) and either use that directly (will probably have to ask users to confirm the year b/c I think different versions of excel use different starting points) or somehow just take the "non-integer" portion and use that as the hours/minutes/seconds ? 

TL;DR - even with it now taking excel, we are still not guaranteed to get the "seconds" portion of the data if the excel formatting is off. can we use the serial number somehow instead?

**Comment by koay9f** [comment]
* Can we get the "seconds" fix you made in when using csv as well?
* Data with separate date & time seems to be stuck on "Cleaning and setting up day data, one moment."
![image.png](https://images.zenhubusercontent.com/5cd48a2af8cffa5a19122d27/7b51d432-5554-42b7-8ca4-0828a6c4cece)

**Comment by shivasaurus** [comment]
It looks like it isn't possible to read seconds from CSV data when Excel formatting has wiped the seconds, since Excel's CSV export will delete the seconds from the CSV file entirely.

**Comment by shivasaurus** [comment]
The time column is now deleted during data cleaning.
